src/limestone-api.ts

```diff
@@ -33,8 +35,8 @@ export default class LimestoneApi {
     arweaveProxy: ArweaveProxy;
   }) {
     this.arweaveProxy = opts.arweaveProxy;
```

Comment:
Is it possible to simplify if as it seems we default this -> opts -> config. Will something like Object.assign(this, opts, config) work?